backend/docker-compose.yaml

```diff
@@ -13,10 +13,16 @@ services:
          DATABASE_USERNAME: strapi
          DATABASE_PASSWORD: strapi
          JWT_SECRET: '${JWT_SECRET}'
+         API_TOKEN_SALT: '${API_TOKEN_SALT}'
       ports:
          - '${HOST_PORT:-1337}:1337'
       volumes:
-         - uploads:/srv/app/public/uploads
+         - uploads:/opt/app/public/uploads
+         - ./config:/opt/app/config
+         - ./src:/opt/app/src
+         - ./package.json:/opt/package.json
+         - ./yarn.lock:/opt/yarn.lock
+         - ./.env:/opt/app/.env
```

Comment:
Why are all of these directories using volumes? It looks like this will end up putting all the code in volumes, instead of using it from the container. And I think, after the first boot, no further changes will take effect on the volumes; I believe they get copies of the data that's present on first boot, but then remain unchanged after that.

Reply:
@andyg0808 
I think this is configured this way because it can support running Strapi in dev mode (although we probably won't need it). Do you want to get rid of this?

Reply:
Do we anticipate any performance problems with keeping this configuration? If not, we can probably leave it alone I think

Reply:
@lithobraking from Docker docs:

<img width="931" alt="image" src="https://user-images.githubusercontent.com/4640135/166489028-9df172b0-b37a-4200-a021-2b0cb15ccd8b.png">

Since the dashboard is currently broken I would merge this as is.. I can always open a PR later for Docker optimizations

Reply:
@dhmacs Agreed. We can worry about it later.